testproject/Assets/Scripts/testproject.asmdef

```diff
@@ -7,7 +7,9 @@
         "Unity.Multiplayer.MLAPI.Prototyping",
         "Unity.Multiplayer.Transport.UTP",
         "Unity.Services.Authentication",
-        "Unity.Services.Core"
+        "Unity.Services.Core",
+        "Unity.Networking.Transport",
```

Comment:
Are we actually referencing `Unity.Networking.Transport` in the test project?

Reply:
Not anymore, that must be a remnant from my initial iteration where I was directly using the `RelayServerData` object in the project-level code.